### PR TITLE
feat: add jsjoeio for talks as contributor

### PR DESCRIPTION
**Pull request by jsjoeio**
**What**:

- add myself (@jsjoeio) as a contributor for docs

**Why**:

@mpeyper [tweeted me](https://twitter.com/michaelpeyper/status/1199122543719747590?s=20) saying I should add myself for [sharing this library](https://twitter.com/michaelpeyper/status/1199122970322452482?s=20) in my custom hooks course on egghead 

**How**:

Using the script `npm run contributors:add` (hadn't seen the CLI version, super nice BTW(

**Checklist**:

- [x] Documentation updated
- [ ] Tests
- [x] Ready to be merged
      
- [x] Added myself to contributors table

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -138,6 +138,15 @@
...
         "doc"
       ]
+    },
+    {
+      "login": "jsjoeio",
+      "name": "JavaScript Joe",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/3806031?v=4",
+      "profile": "https://jsjoe.io",
+      "contributions": [
+        "talk"
```
Comment (mpeyper) [inline .all-contributorsrc]: happy if you're happy, but does `tutorial` fix an egghead course better that `talk`? 
 You can see all the options [here](https://allcontributors.org/docs/en/emoji-key).

**Comment by mpeyper** [comment]
I love your dedication to the PR template!

Let me know on the above question and I'll merge this.

File: .all-contributorsrc (hunk 1 above)
Comment (jsjoeio) [reply .all-contributorsrc]: Ah, I didn't see `tutorial` - that makes more sense. Good catch! Let me update that

**Comment by jsjoeio** [comment]
> I love your dedication to the PR template!

😂 I love PR templates - you have to honor the process!

**Pull request merged**